test-target/storage.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: test-pv
+  labels:
+    role: test
+spec:
+  capacity:
+    storage: 1Gi
```

Comment:
Do we really need 1Gi or less can be fine also?

Reply:
Yes, we can create the PV with less capacity as it is not being used anyway. I will change it to `10Mi` instead.